flink-streaming-java/src/test/java/org/apache/flink/streaming/util/OneInputStreamOperatorTestHarness.java

```diff
@@ -98,7 +99,7 @@ public OneInputStreamOperatorTestHarness(OneInputStreamOperator<IN, OUT> operato
 	public OneInputStreamOperatorTestHarness(
 			OneInputStreamOperator<IN, OUT> operator,
 			ExecutionConfig executionConfig) {
-		this(operator, executionConfig, DefaultTimeServiceProvider.create(Executors.newSingleThreadScheduledExecutor()));
+		this(operator, executionConfig, null);
```

Comment:
This can directly be `DefaultTimeServiceProvider.create(mockTask, Executors.newSingleThreadScheduledExecutor())` instead of the `null` check below.